test/proxy/transparent/TransparentUpgradeableProxy.behaviour.js

```diff
@@ -107,11 +94,6 @@ module.exports = function shouldBehaveLikeTransparentUpgradeableProxy (createPro
             this.receipt = await this.proxy.upgradeToAndCall(this.behavior.address, initializeData, { from, value });
           });
 
-          it('upgrades to the requested implementation', async function () {
```

Comment:
We need to keep this test. We just have to use the new recommended method to retrieve the implementation address of a proxy: reading the EIP-1967 storage slot directly.

I think the same applies to most of the following tests that were removed, can you review them?

Reply:
You are right, as a side note: I'll remove the `storage` section since I use storage to get implementation and admin everywhere, having a specific test for it is redundant.